src/response/cache_dictionary_fetch_response.rs

```diff
@@ -1,20 +1,56 @@
-use std::collections::HashMap;
-
-/// Encapsulates the status of a cache disctionary get operation.
-#[derive(Debug)]
-pub enum MomentoDictionaryFetchStatus {
-    /// Status if the dictionary was found
-    FOUND,
-    /// Status if the dictionary was missing
-    MISSING,
-    ERROR,
+use std::{
+    collections::HashMap,
+    convert::{TryFrom, TryInto},
+};
+
+use crate::MomentoError;
+
+use super::DictionaryPairs;
+
+/// Response for a dictionary fetch operation.
+#[derive(Debug, PartialEq, Eq)]
+pub enum DictionaryFetch {
+    /// The dictionary item existed in the cache
+    Hit { value: DictionaryPairs },
+    /// The dictionary item did not exist
+    Miss,
+}
+
+impl TryFrom<DictionaryFetch> for HashMap<Vec<u8>, Vec<u8>> {
```

Comment:
did you decide you want to keep this?  I'm still kinda in favor of removing this one and just supporting strings for map keys for now

Reply:
will drop